awx/main/conf.py

```diff
@@ -765,6 +765,19 @@
     help_text=_('Indicates whether the instance is part of a kubernetes-based deployment.'),
 )
 
+# Possible values
+# - ''
+# - unique_managed_hosts
+register(
+    'SUBSCRIPTION_USAGE_MODEL',
+    field_class=fields.CharField,
```

Comment:
if it has 2 possible values then `field_class=fields.ChoiceField` would take care of your validation.

Reply:
Right, we talked we that we can even make the 'unique_managed_hosts' default, so we don't have to include installer in this.

the '' strategy (no subscription model strategy) will be covered by the productization flag (that is now hiding the whole subs section) 

Then choice field with ['unique_managed_hosts', 'unique_managed_hosts_monthly'] sounds good.